test/K6/src/data/testdata/delegations/at21testdata.json

```diff
@@ -0,0 +1,38 @@
+{
+    "env": "at23",
+    "org": "ttd",
+    "app": "apps-test",
+    "org1": {
+        "orgno": "910033166",
+        "partyid": "50078191",
+        "dagl": {
+            "userid": "20003434",
+            "partyid": "50012985"
+        }
+    },
+    "org2": {
```

Comment:
Could be nice to have naming of org which describes it a bit more than "org2".
E.g. Whether it's a subunit/mainunit/enk/nuf/revi/regn etc. to make it easier to know which data exists already or if one needs to create a new test data?

Reply:
I needed two different orgs for the tests and just named them org1 and org2. Would mainunit be a better name in this case?